models/constants.py

```diff
@@ -0,0 +1,125 @@
+SEEDS_TO_COUNT = [
+    'kmer8_count',
+    'kmer7_count',
+    'kmer6_count',
+    'kmer6_bulge_count',
+    'kmer6_mismatch_count',
+]
+
+SEED_COUNT_COLUMNS = [
+    'kmer8_count_1', 'kmer8_count_2',
+       'kmer8_count_3', 'kmer8_count_4', 'kmer8_count_5', 'kmer8_count_6',
+       'kmer7_count_1', 'kmer7_count_2', 'kmer7_count_3', 'kmer7_count_4',
+       'kmer7_count_5', 'kmer7_count_6', 'kmer6_count_1', 'kmer6_count_2',
+       'kmer6_count_3', 'kmer6_count_4', 'kmer6_count_5', 'kmer6_count_6',
+       'kmer6_bulge_count_1', 'kmer6_bulge_count_2', 'kmer6_bulge_count_3',
+       'kmer6_bulge_count_4', 'kmer6_bulge_count_5', 'kmer6_bulge_count_6',
+       'kmer6_bulge_or_mismatch_count_1', 'kmer6_bulge_or_mismatch_count_2',
+       'kmer6_bulge_or_mismatch_count_3', 'kmer6_bulge_or_mismatch_count_4',
+       'kmer6_bulge_or_mismatch_count_5', 'kmer6_bulge_or_mismatch_count_6',
+]
+
+TARGETSCAN_COLUMN_TO_SEQUENCE = {
+    'hsa-miR-16-5p': 'TAGCAGCACGTAAATATTGGCG', 
+    'hsa-miR-106b-5p': 'TAAAGTGCTGACAGTGCAGAT', 
+    'hsa-miR-200a-3p': 'TAACACTGTCTGGTAACGATGT', # https://mirbase.org/hairpin/MI0000737
+    'hsa-miR-200b-3p': 'TAATACTGCCTGGTAATGATGA', 
+    'hsa-miR-215-5p': 'ATGACCTATGAATTGACAGAC', 
+    'hsa-let-7c-5p': 'TGAGGTAGTAGGTTGTATGGTT', 
+    'hsa-miR-103a-3p': 'AGCAGCATTGTACAGGGCTATGA'
+}
+
+PSILAC_DATA = {
```

Comment:
Providing links for all sequences, where you took them form, would be great.

Reply:
Adding to the backlog, will be solved when adding the other datasets